Sources/Sentry/Profiling/SentryProfilerSerialization.mm

```diff
@@ -308,15 +308,16 @@
 #    endif // SENTRY_HAS_UIKIT
 )
 {
-    const auto payload
-        = sentry_serializedContinuousProfileChunk(profileID, profileState, metricProfilerState,
-            [SentryDependencyContainer.sharedInstance.debugImageProvider getDebugImagesCrashed:NO],
-            SentrySDK.currentHub
+    const auto chunkID = [[SentryId alloc] init];
+    const auto payload = sentry_serializedContinuousProfileChunk(
+        profileID, chunkID, profileState, metricProfilerState,
```

Comment:
What's the `profileID` here? Is it meant to be `profilerID`?

Reply:
it does get written to `profiler_id` in the payload